core/src/main/resources/hudson/model/Run/console.jelly

```diff
@@ -28,29 +28,24 @@ THE SOFTWARE.
   Displays the console output
 -->
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:t="/lib/hudson">
-  <l:layout title="${it.fullDisplayName} Console">
-    <st:include page="sidepanel.jelly" />
-
-    <l:breadcrumb title="${%Console Output}" />
-
-    <l:main-panel>
-      <j:set var="controls">
-        <a class="jenkins-button" href="consoleText" download="${it.displayName}.txt">
-          <l:icon src="symbol-download" />
-          ${%Download}
-        </a>
-        <l:copyButton ref="out" label="${%Copy}" />
-        <a class="jenkins-button" href="consoleText">
-          ${%View as plain text}
-        </a>
-      </j:set>
-
-      <t:buildCaption controls="${controls}">
-        ${%Console Output}
-      </t:buildCaption>
-
-      <st:include page="console-log.jelly" />
-    </l:main-panel>
-  </l:layout>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout">
+  <j:new className="jenkins.run.ConsoleTab" var="it">
+    <j:arg value="${it}"/>
+  </j:new>
+
+  <l:run-host>
```

Comment:
Unfortunately this doesn't apply to pipeline jobs and needs another PR there:

https://github.com/jenkinsci/workflow-job-plugin/blob/master/src/main/resources/org/jenkinsci/plugins/workflow/job/WorkflowRun/console.jelly

It would be nice to see if we can remove the duplication although looks like there is pipeline specific stuff in there atm